Added info source tab

davidye007 opened the pull request:
Added a flipped bar chart in the info source tab. The user can select either display mode count or %.
Updated subtabs formatting to fit them all on one line.

Merge together with https://github.com/OutdoorRD/trails-viz-data/pull/80.

davidye007 commented:
@sgwinder @EmiliaH Please take a look at the new Info Source tab under Visitor Characteristics. Let me know if you have any questions, doubts, or concerns. Thanks!

Before running locally, make sure you have the corresponding data (updated with info source). 
In the `trails-viz-data` repository terminal, run:
```
git checkout update_chatbot_data_with_infosource
git pull
```

EmiliaH reviewed (changes requested):
Hey @davidye007! Nice work (also I was able to successfully test, don't know what the issue was with the last PR test)! 

I can't seem to find where I wrote down some alternative bin names (it doesn't seem to be in our info source issue https://github.com/OutdoorRD/west-cascades/issues/160#issuecomment-2787823885) or internal meeting notes. But I'd like to make a couple requests

`previous` -> `previous visit`
`found` -> `found/exploring`
`local` -> `local knowledge`

@sgwinder, please let me know if you disagree. 

Also I checked out the data generation here https://github.com/OutdoorRD/vic-flaskbot/pull/254/files.

sgwinder reviewed (approved):
Looks great, and ran successfully for me!

davidye007 commented:
Hey @EmiliaH, that sounds good!

I updated the `chatbot_data_prep` script [PR](https://github.com/OutdoorRD/vic-flaskbot/pull/254) to reflect the info source names you requested. I also updated the `trails-viz-data`  [PR](https://github.com/OutdoorRD/trails-viz-data/pull/80) with the new info source names.
